### PR TITLE
Create new module,  widget text with background

### PR DESCRIPTION
**Pull request by moloudii**

### CODE REVIEW

File: sass/modules/item/_bg_with_text.scss [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        line-height     : 2em;
+        font-weight     : bold;
+    }
+    .explanation{
+        color           : $white;
+        line-height     : 1.5em;
+    }
+    .Price{
+        font-size       : 1.5em;
+        color           : $SUN_FLOWER ;
```
Comment (Mekaeil) [inline sass/modules/item/_bg_with_text.scss]: use the correct variable name. please take a look at the variable file and you can see the variable name example.

File: sass/_variables.scss [hunk 2]
```diff
@@ -47,6 +47,9 @@
     $black          : rgb(000, 000, 000); // #000
     $white          : rgb(255, 255, 255); //#fff
 
+    $SILVER-O20     : rgba(189, 195, 199, 0.2);
```
Comment (Mekaeil) [inline sass/_variables.scss]: use the correct variable name. please take a look at the variable file and you can see the variable name example.

File: sass/modules/item/_bg_with_text.scss [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+        transition      : all 1s;
+    }  
+    .text-glass{
+        width           : auto;
+        margin-left     : 20em;
+        margin-right    : 3em;
+        line-height     : 18.5em;
+    }  
+    .title-card{
+        color           : $CLOUDS;
```
Comment (Mekaeil) [inline sass/modules/item/_bg_with_text.scss]: use the correct variable name.

File: index-_bg_with_text.html [hunk 4]
```diff
@@ -57,10 +57,10 @@
...
                         <span class="title-card">
-                            <p>20% OFF</p>
+                            <p>تخفیف 20%ی</p>
                         </span>
                         <span class="bold-card">
-                            <p>Croatia Cruise Adventure</p>
+                            <p>اعتماد شما سرمایه ماست</p>
                         </span>
                         <span class="explanation">
                             <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ab expedita dolorem atque odit recusandae odit recusandae.</p>
```
Comment (Mekaeil) [inline index-_bg_with_text.html]: Write Persian text

File: sass/_variables.scss [hunk 5]
```diff
@@ -44,9 +44,12 @@
...
     $ASBESTOS       : rgb(127, 140, 141); // #7f8c8d
 
-    $black          : rgb(000, 000, 000); // #000
-    $white          : rgb(255, 255, 255); //#fff
+    $BLACK          : rgb(000, 000, 000); // #000
+    $WHITE          : rgb(255, 255, 255); //#fff
 
+    $SILVER-O20     : rgba(189, 195, 199, 0.2); 
+    $SILVER-O40     : rgba(189, 195, 199, 0.4); 
+    $BLACK-O30      :rgba(0,0,0,0.3);
```
Comment (Mekaeil) [inline sass/_variables.scss]: user the darken and lighten methods in Sass and remove these variables.

File: sass/modules/item/_bg_with_text.scss [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+
+
+    .travel-background{
+        background-image: url("../img/bg-with-image/travel-background.jpg");
+        width           : 100%;
+        height          : 18.5em;
+        background-size : cover;
+    }
+    .glass-div{
+        background-color: $BLACK-O30;
```
Comment (Mekaeil) [inline sass/modules/item/_bg_with_text.scss]: you can use this method:
```
lighten($BLACK, 30%);
```
follow this link: https://falkus.co/2017/05/using-lighten-and-darken-in-sass/

Comment (moloudii) [reply sass/modules/item/_bg_with_text.scss]: I solved it using rgba

**Pull request merged**